docker-compose.yml

```diff
@@ -54,7 +51,6 @@ services:
       DB_HOST: app_db
       WP_URL: 'http://localhost'
       WP_DOMAIN: 'localhost'
-      STRIPE_GATEWAY: 1
```

Comment:
I don't see where these are used in this repo.

Reply:
Ya. . .they're not. Copy pasta from the docker config on the WPGraphQL for WooCommerce repo. 🤦‍♂️

Reply:
No worries.